compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/ManifestBuilder.java

```diff
@@ -257,49 +255,53 @@ private PackageDescriptor getPackageDescriptor(TomlTableNode tomlTableNode) {
         String name;
         String version;
 
+        String errorMessage = "missing package information in Ballerina.toml, values derived as org: '"
```

Comment:
Here is my suggestion:

```
Missing table `[package]` in `Ballerina.toml`. Defaulting to:
[package]
org = "foo"
name = "bar"
version = 1.0.0
```

Reply:
Refactored via https://github.com/ballerina-platform/ballerina-lang/pull/36174/commits/48fba38888323c8a03144c32f77884571964360f